nntrainer/layers/bn_layer.cpp

```diff
@@ -256,6 +256,14 @@ void BatchNormalizationLayer::setBatch(RunLayerContext &context,
                                        unsigned int batch) {
   context.updateTensor(wt_idx[BNParams::deviation], batch);
   context.updateTensor(wt_idx[BNParams::t_full], batch);
+
+  /// reset divider
+  divider = 1;
+  auto input_dim = context.getInput(0).getDim();
+  input_dim.batch(batch);
```

Comment:
This is already done by layer node and can be skipped.

Reply:
Please note that, input_dim.batch is not updated when this function is being called. We might want to update inputdims before coming here later.